pkgs/build-support/cc-wrapper/add-clang-cc-cflags-before.sh

```diff
@@ -2,7 +2,11 @@ needsTarget=true
 
 for p in "${params[@]}"; do
     case "$p" in
-    -target | --target=*) needsTarget=false ;;
+    -target | --target=*)
+        needsTarget=false
+
+        echo "Warning: supplying the --target argument to a nix-wrapped compiler probably won't work correctly - cc-wrapper is not designed with multi-target compilers in mind. You may want to use an un-wrapped compiler instead." >&2
```

Comment:
Could we soften this a little bit?  I think we'll come up with a way to make it work, at least for use cases we care about, so maybe we could emphasize that _currently_ it might not work correctly?